### PR TITLE
Allow HSETNX to receive multiple field-value pairs like HSET

### LINKED ISSUE #1294: Perform HSETNX with multiple values, just like HSET
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Motivation

With HSET we can add a multiple key/field at once. But HSETNX, that's insert value only if they does not yet exist, use only one key/field pair at once. 

Could we changed HSETNX for usage with multiple values? 

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by torwig**
Close #1294

### CODE REVIEW

File: src/commands/cmd_hash.cc [hunk 1]
```diff
@@ -44,10 +44,22 @@ class CommandHGet : public Commander {
...
+    if (args.size() % 2 != 0) {
+      return {Status::RedisParseErr, errWrongNumOfArguments};
+    }
+    return Commander::Parse(args);
+  }
+
   Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    std::vector<FieldValue> field_values;
+    for (size_t i = 2; i < args_.size(); i += 2) {
+      field_values.emplace_back(FieldValue{args_[i], args_[i + 1]});
```
Comment (PragmaTwice) [inline src/commands/cmd_hash.cc]:
```suggestion
      field_values.emplace_back(args_[i], args_[i + 1]);
```

Comment (PragmaTwice) [reply src/commands/cmd_hash.cc]: BTW I think it is more proper to put the for loop into `Parse`.

Comment (torwig) [reply src/commands/cmd_hash.cc]: @PragmaTwice Thank you for your reasonable suggestions.

Comment (torwig) [reply src/commands/cmd_hash.cc]: Done

**Review by PragmaTwice — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Comment by PragmaTwice** [comment]
Thanks @torwig, merging...

**Pull request merged**